src/api/window/window.ts

```diff
@@ -5,12 +5,101 @@ import BoundsChangedReply from './bounds-changed';
 import Animation from './animation';
 import { Application } from '../application/application';
 import Transport from '../../transport/transport';
+//import { initialOptions, windowExists, createChildWindow } from '../../environment/openfin-renderer-api';
+//import { windowExists } from '../../environment/openfin-renderer-api';
+import * as _ from 'underscore';
+
+declare var fin: any;
```

Comment:
Would like to use environment here instead of globals:
https://github.com/HadoukenIO/js-adapter/blob/master/src/environment/openfin-env.ts

And handle the unsupported nodejs Create case.

Reply:
Really good point. I just changed it.